pycbc/inference/recal.py

```diff
@@ -0,0 +1,102 @@
+# Module to hold recalibration stuff used during parameter estimation
```

Comment:
Don't add to ``inference``, this is much more general than just inference stuff.

Make a new subpackage ``pycbc.calibration``. And add a ``pycbc.calibraion.adjust`` or whatever you call it.

Reply:
Also don't make this comments, use the ``"""`` so it shows up in the webpages and IDE.